documentation/source/newsfragments/1495.removal.rst

```diff
@@ -1,5 +1,4 @@
-The makefile variables
-:make:var:`COCOTB_NVC_TRACE`, :make:var:`VERILATOR_TRACE`
-that were not supported for all simulators have been deprecated.
-Using them prints a deprecation warning and points to the documentation section
+The makefile variable :make:var:`VERILATOR_TRACE`
+that was not supported for all simulators has been deprecated.
+Using it prints a deprecation warning and points to the documentation section
```

Comment:
This is still pretty wordy.

```suggestion
:make:var:`VERILATOR_TRACE` has been deprecated. Use :make:var:`SIM_ARGS` to pass the `--trace` option instead.
```
Might have gotten that second sentence wrong.

We are going to have one chunky 1.4 changelog if we don't keep these more brief. The PR is linked, users can look at them if they want more detail.

Reply:
This newsfragment will over time collect several deprecations (#1495 is a collection ticket), so I believe the original wording will be small compared to the number of deprecated variables it will mention.